modules/settings/styles/settings.module.css

```diff
@@ -15,4 +15,163 @@
   flex-grow: 1;
   padding: 20px;
   overflow-y: auto;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.form {
+  width: 100%;
+  max-width: 600px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.mainFormGroup {
+  width: 100%;
+  padding: 1rem 5rem;
+}
+
+.formGroup {
+  margin-bottom: 20px;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+}
+
+.inputContainer {
+  display: flex;
+  align-items: center;
+}
+
+.formGroup label {
+  flex: 0 0 40%;
+  text-align: right;
+  margin-right: 20px;
+  color: black;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 2vh;
+}
+
+.formGroup input,
+.formGroup select {
+  flex: 0 0 60%;
+  padding: 8px;
+  border: 1px solid #ccc;
+  border-radius: 4px;
+  height: 6vh;
+  color: black;
+  font-size: 2vh;
+  font-family: 'Montserrat', sans-serif;
+}
+
+.errorMessage {
+  color: red;
+  font-size: 2vh;
+  margin-top: 5px;
+  margin-left: calc(40% + 20px);
+  font-family: 'Montserrat', sans-serif;
+}
+
+.socialLogins {
+  padding: 3vh 12.5vh 1vh 12.5vh;
+}
+
+.oauthButton {
```

Comment:
Seems oauth buttn canb e a specific component with their own style.maybe social logins (to reduces this css)

Reply:
This oauth button is vastly different from the one used in login/register and it's only used here hence no need to make it reusable